src/virt.c

```diff
@@ -578,15 +614,26 @@ lv_read (void)
                     &stats, sizeof stats) != 0)
             continue;
 
+	char *type_instance = NULL;
+	if( blockdevice_format_basename && blockdevice_format == source ) //valid only if we use "source" (full path to the device)
```

Comment:
Regarding your comment for the section before this: "I still see the if-else blocks. Did you forget to push the commit switching to the IS_TRUE() macro?"
A: This is the part of the code, which parses the config options. From what I can see from the other plugins the true/false options are coming as strings and I am supposed to use strcasecmp.
Is this correct ?